net/libfab/libfab.c

```diff
@@ -920,14 +1012,13 @@ static int libfab_passive_ep_create(struct m0_fab__ep *ep,
 	M0_ALLOC_PTR(ep->fep_listen);
 	if (ep->fep_listen == NULL)
 		return M0_RC(-ENOMEM);
-	M0_ALLOC_PTR(ep->fep_listen->pep_tx_ep);
-	if (ep->fep_listen->pep_tx_ep == NULL)
-		return M0_RC(-ENOMEM);
-	M0_ALLOC_PTR(ep->fep_listen->pep_rx_ep);
-	if (ep->fep_listen->pep_rx_ep == NULL)
+	M0_ALLOC_PTR(ep->fep_listen->pep_aep);
+	if (ep->fep_listen->pep_aep == NULL)
 		return M0_RC(-ENOMEM);
 
 	pep = ep->fep_listen;
+	ep->fep_listen->pep_aep->aep_rxep = NULL;
+	ep->fep_listen->pep_aep->aep_txep = NULL;
 
 	if (strlen(ep->fep_name.fen_port) != 0){
```

Comment:
An space is needed before the opening brace:
```
if (strlen(ep->fep_name.fen_port) != 0) {
```

Reply:
And please check this for all other places.

Reply:
Done